sdks/python/apache_beam/options/pipeline_options_validator.py

```diff
@@ -174,6 +179,16 @@ def validate_cloud_options(self, view):
       if not view.job_name:
         errors.extend(self._validate_error(
             'Existing job name must be provided when updating a pipeline.'))
+    if view.transform_name_mapping:
```

Comment:
Can we also confirm in here that the job is streaming and update.

Reply:
added streaming option validation below.